final_prep/mod_zero_hero.rb

```diff
@@ -1,84 +1,204 @@
-# Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
+# Challenge - See if you can follow the instructions and complete the exercise
+#in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
+hero_name = "Superman"
+special_ability = "super human strength"
 
 # Declare two variables - greeting AND catchphrase
-#   greeting should be assigned to a string that uses interpolation to include the hero_name
-#   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+#   greeting should be assigned to a string that uses interpolation to include
+#   the hero_name
+greeting = "Hello, I am #{hero_name}!"
+#   catchphrase should be assigned to a string that uses interpolation to
+#   include the special_ability
+catchphrase = "Don't worry! I will use my #{special_ability} to rescue you."
 
-# Declare two variables - power AND energy - set to integers
 
+# Declare two variables - power AND energy - set to integers
+power = 8
+energy = 10
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
+full_power = power * 500
 #   full_energy should add 150 to your current energy
+full_energy = energy + 150
 
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
+is_human = false
+identify_concealed = true
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
+arch_enemies = ["Lex Luthor", "Darkseid", "Brainiac"]
 #   sidekicks should be an array of at least 3 different sidekick strings
+sidekicks = ["Superboy", "Lois Lane", "Supergirl"]
 
 # Print the first sidekick to your terminal
+puts sidekicks[0]
 
 # Print the last arch_enemy to the terminal
+puts arch_enemies[2]
 
 # Write some code to add a new arch_enemy to the arch_enemies array
-
+arch_enemies << "Black Zero"
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
+puts arch_enemies
 
 # Remove the first sidekick from the sidekicks array
+sidekicks.delete_at(0)
 
 # Print the sidekicks array to terminal to ensure you added a new sidekick
-
-# Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
-#   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
-#   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
+puts sidekicks
+
+# Create a function called assess_situation that takes three arguments -
+# danger_level, save_the_day, bad_excuse
+def assess_situation(danger_level, save_the_day, bad_excuse)
+  #   - danger_level should be an integer
+  danger_level = 9
```

Comment:
Your `assess_situation` function is taking in an argument called `danger_level` (this is a function argument or placeholder, not a string), and yet you're setting a variable with the same name on L57 to an arbitrary integer.  You don't need to declare the var on L57 since `danger_level` is already getting passed in as a function argument.